src/main/kotlin/com/approvers/devlazaApi/model/Projects.kt

```diff
@@ -28,6 +31,7 @@ data class ProjectPoster(
 )
 
 @Entity
+@Table(indexes = [Index(name="sites_index", columnList="PROJECT_ID, URL", unique=true)])
```

Comment:
```suggestion
@Table(indexes = [
    Index(columnList="url"),
    Index(columnList="project_id")
])
```
project_idはuniqueならunique付けといて

Reply:
いや、合わせたらuniqueだが単体はuniqueじゃない